frontend/e-learning-fe/src/components/login.js

```diff
@@ -120,37 +126,29 @@ function Login() {
                   setPassword(e.target.value);
                   checkFields();
                 }}
+                disabled={isLoading}
               />
             </div>
 
-            {isDisable === true ? (
-              <button
-                type="submit"
-                class="w-full text-white bg-primary-600 hover:bg-primary-700 focus:ring-4 
-                    focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center 
-                    dark:bg-blue-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800"
-                onClick={handleLogin}
-              >
-                Sign in
-              </button>
-            ) : (
-              <button
-                type="submit"
-                class="w-full text-white bg-primary-600 hover:bg-primary-700 focus:ring-4 
-                    focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center 
-                    dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800"
-                disabled
-              >
-                Sign in
-              </button>
-            )}
+            <button
+              type="submit"
+              class={
+                isDisableSignInButton
+                  ? "w-full text-white bg-primary-600 hover:bg-primary-700 focus:ring-4 focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center dark:bg-gray-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800"
+                  : "w-full text-white bg-primary-600 hover:bg-primary-700 focus:ring-4 focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center dark:bg-blue-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800"
```

Comment:
only these part here is different and the rest is the same right?
disabled: `dark:bg-gray-600`
enabled: `dark:bg-blue-600`

do we have a specific reason why we repeated the parts that are the same?

Reply:
Opo pero parang as one kasi sya heheh di ko po sila mahiwalay

Reply:
sorry last confirmation, this does not work? does it raise an error po ba? or just straight up not working pag ganito :bow: 
```
className = {`w-full text-white bg-primary-600 hover:bg-primary-700 focus:ring-4 focus:outline-none focus:ring-primary-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center dark:hover:bg-primary-700 dark:focus:ring-primary-800 ${isDisabled ? 'dark:bg-blue-600' : 'dark:bg-gray-600'}`}
```

Reply:
Working na sya sir mali pala yung ginawa ko, ngayon ko lang po nalaman yang ginawa mo hehehe push ko na po saglit after testing

Reply:
Done sir